geomesa-hbase/geomesa-hbase-tools/src/main/resources/org/locationtech/geomesa/hbase/tools/ingest-libjars.list

```diff
@@ -7,6 +7,9 @@ geomesa
 gson
 gt
 hbase
+hbase-client
+hbase-server
+hbase-protocol
```

Comment:
these are fine - the `hbase` prefix should pick them all up if they're in the same directory, but this would cover cases where they aren't.